<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Automate_Widget</class>
 <widget class="QWidget" name="Automate_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(171, 114, 0);
color: rgb(255, 255, 255);
font-size: 10pt;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Automation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Record Mode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_rec_mode">
         <item>
          <property name="text">
           <string>Add</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Overwrite</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_clear">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 157, 157);
font-size: 10pt;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>CLEAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_rec">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(157, 0, 0);
font-size: 10pt;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>REC</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hold</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_hold"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ramp Up</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_ramp_up"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ramp Down</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBox_ramp_down"/>
     </item>
     <item row="1" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_key">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_enabled">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(162, 6, 6);
font-size: 10pt;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
